packages/flutter/test/material/date_range_picker_test.dart

```diff
@@ -268,12 +268,10 @@ void main() {
     });
   });
 
-  testWidgets('Long helpText cant not cutoffs the save button', (WidgetTester tester) async {
+  testWidgets('Long helpText does not cutoff the save button', (WidgetTester tester) async {
     helpText = 'long helpText' * 100;
-    saveText = 'make it so';
     await preparePicker(tester, (Future<DateTimeRange?> range) async {
-      expect(find.text(helpText!), findsOneWidget);
-      expect(find.text(saveText!), findsOneWidget);
+      expect(tester.takeException(), null);
```

Comment:
I think it would be better to have the check you had before and this one. So both this `.takeException()` and the check for `helpText` and `saveText`.

Reply:
I believe the check for `helpText` and `saveText` is already there; the most recent commit reverts the change and adds a new test.

Here's what the branch looks like currently:

https://github.com/flutter/flutter/blob/3c26c7d2aa29917c901bec291a21755d43214195/packages/flutter/test/material/date_range_picker_test.dart#L261-L277

Reply:
Oh, that makes sense! 
```suggestion
      expect(tester.takeException(), null);
      expect(find.text(helpText!), findsOneWidget); 
      expect(find.text(saveText!), findsOneWidget); 
```

Reply:
@MitchellGoodwin Thank you for the review, sorry for my misunderstanding, I have changed the test.